elastalert/test_rule.py

```diff
@@ -402,6 +402,13 @@ def run_rule_test(self):
         overwrites = {
             'rules_loader': 'file',
         }
+
+        # Set arguments that ElastAlerter needs
+        args.verbose = args.alert
+        args.debug = not args.verbose
```

Comment:
I'm just a random person from the internet, not a project maintainer, but how do you feel about the following?

```suggestion
        args.debug = not args.alert
```

I realize it's functionally equivalent to your implementation, but feels a bit more consistent with line 232, https://github.com/Yelp/elastalert/blob/ecf40ad31530c34c522d2f8162f6df8f4b32fb1c/elastalert/test_rule.py#L232

Also the conditional on lines 304-307, https://github.com/Yelp/elastalert/blob/ecf40ad31530c34c522d2f8162f6df8f4b32fb1c/elastalert/test_rule.py#L304-L307

It also relates more directly to the help text, 
https://github.com/Yelp/elastalert/blob/ecf40ad31530c34c522d2f8162f6df8f4b32fb1c/elastalert/test_rule.py#L359